.github/workflows/triage_rotation.yml

```diff
@@ -0,0 +1,40 @@
+name: Assign Issues for Triage 
+
+on:
+  issues:
+    types:
+      - opened
+      
+env:
+ GH_TOKEN: ${{ github.token }}
+
+jobs:
+  assign_issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Calculate assignee
+        id: calculate_assignee
+        run: |
+          USERS=("jrhemstad" "jrhemstad" "jrhemstad" "jrhemstad" "jrhemstad")
```

Comment:
Yeah, we can probably combine these together to create an action that does exactly what we want (I don't imagine random assignment is the goal haha)

- Queries team members that are in a specific team
   - do we need a triage team? I can envision a world where other teams could include people we don't want on triage duty
- Uses the logic from the original action to assign, just having built the list and qty of members from the above query instead of hard-coded

Reply:
Did we want to try to cover that in this PR, or hard-code for now and update in an upcoming GHA automation PR?

Or, we could shelf this entirely until the automation PR since right now it's just 'assign to jake', but I like having it here since the other ones might be hard, and this one just takes some minor hard-coding changes to get working.

Reply:
Hard code for now and leave the team assignment for future work imo.